input/core/src/commonMain/kotlin/majestic/editor/body/chunks/lists/Items.kt

```diff
@@ -0,0 +1,4 @@
+package majestic.editor.body.chunks.lists
+
+
+data class Items(val type: Type, val items: MutableList<String>)
```

Comment:
Instead of having an `Items` class, change the Lists class into something like this

```
data class Lists(
    override val uid: Int,
    val type: Type,
    val items: List<String>
)
```

I don't think you need the extra `Items` class